luigi/parameter.py

```diff
@@ -223,6 +227,23 @@ def parse(self, x):
         """
         return x  # default impl
 
+    def parse_list(self, xs):
```

Comment:
If you happen to know any reading reference with better conventions. I'll be super interested to know. :)